### DISCUSSION TITLE
send books by email

### DISCUSSION THREAD

**Discussion by Helvio88** [gotson/komga#1522] [Ideas]
### Describe your suggested feature

Now that Komga has ePub support, it would be very nice if 2 things were added:

1. A user could have one (or many?) eReader email addresses, so Komga would know where to send the ePub files for eReader availability.
  a. Alternatively, instead of a User setting, an entire new entity could exist: devices. Not sure what the best path would be.
2. Komga should have some basic SMTP client settings, in order to actually deliver the ePub to the device's email.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by solarflair** [comment]
I agree this would be a very useful feature, for Kindle users especially - it's the most popular eink reader brand in the US and possibly the world, and the most convenient way to sideload books to a Kindle reader is by email. And Komga wouldn't even need to support format conversion, since Amazon's email to Kindle feature will now convert epubs on the fly.
